### DISCUSSION TITLE
倍速播放需要更长的提前缓冲

### DISCUSSION THREAD

**Discussion by tfl117** [the1812/Bilibili-Evolved#3564] [功能建议 / Ideas]
首先声明，我的宽带是1000m，测速附上
![QQ图片20220803174021](https://user-images.githubusercontent.com/48679972/182577621-67b56153-46be-44ee-af74-f091c11dc73b.png)
我观看视频有倍速的习惯，我相信大家都常用倍速
b站有个问题，就是视频提前缓冲的量比较小，1倍速播放时没问题
但是1.5-2倍速以及更高倍速时，提前缓冲量较小导致视频卡顿的问题频繁出现。尤其1080p和4k，大会员体验极差
b站当前的缓冲策略在已缓冲内容即将播完时才缓冲下一段，但是每一段都缓冲的太少了，
导致倍速播放时不能及时缓冲视频内容，每段缓冲之间都会存在卡顿，尤其是视频卡音频继续播的情况尤为常见
希望加个组件加大缓冲时长并提早进行缓冲已解决卡顿问题
或者直接缓冲全部也行

**Comment by Geeyun-JY3** [comment]
请善用搜索：#1809、#2243、#2300，其中 #2300 已明确拒绝

**Reply by tfl117** [reply]
既然我不是第一个提出这个问题的，那不是说明有其他人也遇到这个问题了么
是技术问题提难以解决么？解决不了就算了

**Reply by the1812** [reply]
难以解决, 需要高倍速还是建议下载到本地
